src/rotten-tomatoes/SpoilTomatoes.js

```diff
@@ -2,41 +2,40 @@ class SpoilTomatoes {
 
     constructor(grid) {
         this.grid = grid;
-        this.colLength = 0;
-        this.rowLength = 0;
         this.getAdjacents = this.getAdjacents.bind(this)
+        this.getRottenTomatoes = this.getRottenTomatoes.bind(this)
     };
 
     getRottenTomatoes() {
-        const locationOfSpoiltTomatoes = [];
+        const spoiltTomatoesLocations = [];
         let numGoodTomatoe = 0;
         for (let row = 0; row < this.grid.length; row++) {
-            this.rowLength = this.grid.length - 1;
             for (let col = 0; col < this.grid[row].length; col++) {
```

Comment:
With the assumption that the rowSize and columnSize is fixed for this grid, computing this every iterations can be avoided.

Reply:
Okay, I wrote the code this way with a consideration to a senerio where a jagged array way given. i'll make the changes